Add note using CA issuer as intermediate (#499)
cert-manager/website: content/en/docs/configuration/ca.md

```

> > Note: In case your CA issuer represents an intermediate, ensure that `tls.crt`
> > contains the issuing CA certificate _bundle_, in the correct order of signing:
> > `issuing -> intermediate(s) -> <root>`. The root (self-signed) CA certificate is
> > optional, but adding it will ensure that the correct CA certificate is stored in 
> > the `Certificate` secret with key `ca.crt`. When using CA issuer, cert-manager will
> > put the last certificate in the certificate chain configured into client `ca.crt`. 

Reviewer: ```suggestion

> Note: If your issuer represents an intermediate, ensure that `tls.crt` contains
> the issuer's full chain in the correct order: `issuer -> intermediate(s) -> root`.
> The root (self-signed) CA certificate is optional, but adding it will ensure that
> the correct CA certificate is stored in the secrets for issued `Certificate`s under
> the `ca.crt` key.
```

Does this wording work? (I also added an extra line break)

Author: Works for me, but I wonder if anyone would like to know what happens if they don't include the root? Strictly speaking, the only mandatory in issuer `tls.crt` is the issuing CA certificate?

Reviewer: ```suggestion

> Note: If your issuer represents an intermediate, ensure that `tls.crt` contains
> the issuer's full chain in the correct order: `issuer -> intermediate(s) -> root`.
> The root (self-signed) CA certificate is optional, but adding it will ensure that
> the correct CA certificate is stored in the secrets for issued `Certificate`s under
> the `ca.crt` key. If you fail to provide a complete chain, it might not be possible
> for consumers of issued `Certificate`s to verify whether they're trusted.
```

fair point; how about this?

Author: Better! Thanks!
